### DISCUSSION TITLE
style/remove-page报错，无法生成文档

### DISCUSSION THREAD

**Discussion by SparkZu** [note286/xduts#189] [Q&A] [answered]
/paper.tex:15: LaTeX Error: The key 'xdu/style/remove-page' is unknown and is being
(LaTeX)        ignored.
![image](https://github.com/note286/xduts/assets/71439539/ad70b099-06fa-4e07-b191-8bcf698c2ae6)

**Comment by note286** [comment, marked as answer]
由于教务处没有给出查重的具体要求，因此`xduugthesis`没有实现`remove-page`功能，目前建议暂时使用福昕阅读器来删除PDF页面。

**Reply by SparkZu** [reply]
好的好的，感谢回复
